lib/obfuscator.js

```diff
@@ -0,0 +1,30 @@
+const JavaScriptObfuscator = require('javascript-obfuscator');
+const path = require('path');
+const fs = require('fs');
+const glob = require('glob');
+const cwd = process.cwd();
+
+// checks for file named "obfuscate" with new line separated records, each record is a glob pattern for matching js files which should be obfuscated
+module.exports = {
+    obfuscate: function(done) {
+        if (fs.existsSync(path.join(cwd, './obfuscate'))) { // check for file
```

Comment:
maybe name the file .obfuscate, as it appears to be configuration as other dot files